src/rvs_util.cpp

```diff
@@ -180,3 +181,68 @@ bool fetch_gpu_list(int hip_num_gpu_devices, map<int, uint16_t>& gpus_device_ind
     return amd_gpus_found;
 }
 
+
+int display_gpu_info(){
```

Comment:
Can we move this function to gpu utils file ?

Reply:
I thought of that but then it is not exactly a gpu util but something that uses gpu utilities(like dom2node etc) to do something specific to how RVS wants to display the gpus. Hence kept it in rvs_util

Reply:
Ok